installer-build/payload/ai_sperm_analysis/analyze_media.py

```
@@ -0,0 +1,517 @@
+                    analyzed_video_path, heatmap_image_path, total_tracks_detected,
+                    valid_tracks_count, tracking_duration_seconds, frames_analyzed,
+                    detection_accuracy_percent, comments, qc_status
+                ) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+            """, data)
+            
+            conn.commit()
+            conn.close()
+            
+            print("✅ تم حفظ النتائج في قاعدة البيانات")
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)